### ISSUE TITLE
Warning when using `iseed` & `parallel="snow"`: '.Random.seed' is not an integer vector but of type 'double', so ignored

### ISSUE THREAD

**Issue by isaactpetersen** [yrosseel/lavaan#242]
I am trying to estimate a bootstrapped model with parallel estimation and set a seed for reproducibility. However, I receive the following warning when setting `iseed` with `parallel="snow"`, which suggests that my seed is being ignored:

```
'.Random.seed' is not an integer vector but of type 'double', so ignored
```

Here is a minimal reproducible example:

``` r
library("lavaan")
#> Warning: package 'lavaan' was built under R version 4.2.1
#> This is lavaan 0.6-12
#> lavaan is FREE software! Please report any bugs.

HS.model <- '
  visual  =~ x1 + x2 + x3
  textual =~ x4 + x5 + x6
  speed   =~ x7 + x8 + x9
'

RNGkind("L'Ecuyer-CMRG")
RNGkind()
#> [1] "L'Ecuyer-CMRG" "Inversion"     "Rejection"

fit <- cfa(
  HS.model,
  data = HolzingerSwineford1939,
  se = "bootstrap",
  bootstrap = 100,
  parallel = "snow",
  iseed = 123)
#> Warning in RNGkind(): '.Random.seed' is not an integer vector but of type
#> 'double', so ignored

sessionInfo()
#> R version 4.2.0 (2022-04-22 ucrt)
#> Platform: x86_64-w64-mingw32/x64 (64-bit)
#> Running under: Windows 10 x64 (build 19044)
#> 
#> Matrix products: default
#> 
#> locale:
#> [1] LC_COLLATE=English_United States.utf8 
#> [2] LC_CTYPE=English_United States.utf8   
#> [3] LC_MONETARY=English_United States.utf8
#> [4] LC_NUMERIC=C                          
#> [5] LC_TIME=English_United States.utf8    
#> 
#> attached base packages:
#> [1] stats     graphics  grDevices utils     datasets  methods   base     
#> 
#> other attached packages:
#> [1] lavaan_0.6-12
#> 
#> loaded via a namespace (and not attached):
#>  [1] rstudioapi_0.14   knitr_1.40        magrittr_2.0.3    mnormt_2.1.0     
#>  [5] pbivnorm_0.6.0    R.cache_0.16.0    rlang_1.0.6       fastmap_1.1.0    
#>  [9] fansi_1.0.3       stringr_1.4.1     styler_1.7.0      highr_0.9        
#> [13] tools_4.2.0       parallel_4.2.0    xfun_0.33         R.oo_1.25.0      
#> [17] utf8_1.2.2        cli_3.4.1         withr_2.5.0       htmltools_0.5.3  
#> [21] yaml_2.3.5        digest_0.6.29     tibble_3.1.8      lifecycle_1.0.2  
#> [25] purrr_0.3.4       R.utils_2.12.0    vctrs_0.4.1       fs_1.5.2         
#> [29] glue_1.6.2        evaluate_0.16     rmarkdown_2.16    reprex_2.0.2     
#> [33] stringi_1.7.8     compiler_4.2.0    pillar_1.8.1      R.methodsS3_1.8.2
#> [37] stats4_4.2.0      pkgconfig_2.0.3
```

<sup>Created on 2022-09-25 with [reprex v2.0.2](https://reprex.tidyverse.org)</sup>

**Comment by yrosseel** [comment]
Indeed. This has been fixed in the github version (dev 0.6-13). Could you try with the dev version? If there are still issues related to the 'iseed' argument, let me know!

**Comment by isaactpetersen** [comment]
Yes, I verified that it works with the dev version. I will close the issue. Thanks!
